cmd/neofs-cli/modules/object/delete.go

```diff
@@ -28,53 +28,67 @@ func initObjectDeleteCmd() {
 	flags := objectDelCmd.Flags()
 
 	flags.String(commonflags.CIDFlag, "", commonflags.CIDFlagUsage)
-	flags.String(commonflags.OIDFlag, "", commonflags.OIDFlagUsage)
+	flags.StringSlice(commonflags.OIDFlag, nil, commonflags.OIDFlagUsage)
 	flags.Bool(binaryFlag, false, "Deserialize object structure from given file.")
 	flags.String(fileFlag, "", "File with object payload")
+
+	_ = objectDelCmd.MarkFlagRequired(commonflags.CIDFlag)
```

Comment:
Separate thing?

Reply:
it was not possible to implement multiple objects without this change so it is the same change to me, but ok, one more commit has landed